Wancer/Wancer/Models/Group.swift

```diff
@@ -17,9 +17,8 @@ class Group: Identifiable {
     var name: String
     
     @Relationship(deleteRule: .nullify)
-    var users: [User]
-    
-    @Relationship(deleteRule: .cascade)
+    var users: [User]?
+    @Relationship(deleteRule: .cascade, inverse: \Message.group)
```

Comment:
In a parent to children relationship, inverse is automatically assigned. But if its works, we can make this change